experiments/ClimaEarth/setup_run.jl

```diff
@@ -209,8 +210,7 @@ function setup_and_run(config_dict::AbstractDict)
     #and `dir_paths.checkpoints`, where restart files are saved.
     =#
 
-    COUPLER_OUTPUT_DIR = joinpath(output_dir_root, job_id)
-    dir_paths = Utilities.setup_output_dirs(output_dir = COUPLER_OUTPUT_DIR, comms_ctx = comms_ctx)
+    dir_paths = Utilities.setup_output_dirs(output_dir = output_dir_root, comms_ctx = comms_ctx)
```

Comment:
It would be good to avoid changing the output paths if possible

Reply:
Do we need to have a different `COUPLER_OUTPUT_DIR` and `output_dir_root`?

Reply:
No - this is fine with the change you made in `arg_parsing.jl`!